includes/Modules/Ads/AMP_Tag.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Ads\AMP_Tag
+ *
+ * @package   Google\Site_Kit\Modules\Ads
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Ads;
+
+use Google\Site_Kit\Core\Modules\Tags\Module_AMP_Tag;
+use Google\Site_Kit\Core\Tags\Tag_With_Linker_Interface;
+use Google\Site_Kit\Core\Util\Method_Proxy_Trait;
+use Google\Site_Kit\Core\Tags\Tag_With_Linker_Trait;
+
+/**
+ * Class for AMP tag.
+ *
+ * @since  n.e.x.t
+ * @access private
+ * @ignore
+ */
+class AMP_Tag extends Module_AMP_Tag implements Tag_With_Linker_Interface {
+
+	use Method_Proxy_Trait, Tag_With_Linker_Trait;
+
+	/**
+	 * Sets the current home domain.
+	 *
+	 * @since 1.118.0
```

Comment:
Everything in this class should be `n.e.x.t` since it is a new class :)

Reply:
```suggestion
	 * @since n.e.x.t
```